assets/src/util/math.ts

```diff
@@ -7,3 +9,26 @@
  */
 export const clamp = (value: number, min: number, max: number): number =>
   Math.min(Math.max(min, value), max)
+
+/**
+ * Finds the element in {@link positions} that is closest to {@link point}
+ * @param positions List of coordinates to search
+ * @param point Point of reference to check distance to
+ * @returns The closest point in {@link positions} to {@link point}
+ */
+export const closestPosition = (
+  positions: LatLngExpression[],
+  point: LatLng
+) => {
+  const positionsByDistance = positions
+    .map((currentPosition, index) => ({
+      distance: point.distanceTo(currentPosition),
```

Comment:
**praise:** nice, I didn't realize there was a handy method you could use for distance without needing to reference the map object.